src/middlewared/middlewared/etc_files/local/proftpd.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+async def render(service, middleware):
```

Comment:
We are interacting with the filesystem, do you think we should perhaps consider a synchronous method ?